.yarnclean

```diff
@@ -0,0 +1 @@
+
```

Comment:
Did you run yarn autoclean? I don't think you needed to tbh. It seems like it can cause issues unless we're having issues with unnecessary node modules... [autoclean](https://yarnpkg.com/lang/en/docs/cli/autoclean/ )

That being said do we have unnecessary modules as issue lol?

Reply:
It came when I was running `yarn test`. I must be in the script